pkg/lib/cockpit-rsync-plugin.js

```diff
@@ -0,0 +1,31 @@
+const child_process = require("child_process");
+
+module.exports = class {
+    constructor(options) {
+        if (!options)
+            options = {};
+        this.dest = options.dest;
```

Comment:
If this is not given, it looks like the code would just happily rsync into root's home directory. There should be a check that it's not empty.

Also, it would be kind of nice to not *have* to specify it -- it's always `/usr/share/cockpit/<manifestname>`, but indeed at this point that is hard to figure out. So let's keep it explicit for now.

Reply:
Hmm we could always specify "/usr/share/cockpit" as base path in the plugin, as everything should live under it correct?

Then we can go back to what I originally did with `packagename` and append that, if not set it should work for cockpit, what do you think?

Reply:
Not sure what you originally did, but in cockpit itself we have `section`, and in c-podman and friends I suppose we can just hardcode the name.

Reply:
I mean we can change the code to `this.dest = "/usr/share/cockpit/" + dest`. That could in theory avoid hardcoding it in cockpit-machines as we can parse the package.json:
```
cat package.json | jq .name
"machines"
```

Reply:
That's a nice idea -- and parsing JSON from JS is simple. Then we'd only need to override it for cockpit itself, but it would DTRT for starter-kit and friends.